.env

```diff
@@ -0,0 +1 @@
+REACT_APP_BACK_END_URL=https://inviticus-staging.herokuapp.com
```

Comment:
Thanks for creating a separate for our backend URL but I think if it's better to create a new file inside our actions folder and import it where necessary. This is because defining it in .env will prevent others from using it since that file is ignored while commiting. Thanks